src/seth/libexec/seth/seth-source

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+### seth-source -- fetch the contract source from etherscan
```

Comment:
The name `seth-source` makes me think that `source` is a verb. I would rename this to `seth-get-source` or something, to avoid confusion.

Reply:
Isn't sourcing something like fetching the source of it? :)

Reply:
Hahah I don’t know.. not in my mind’s English vocabulary at least!

Reply:
`seth source` is a command for sourcing only the finest smart contract codes from Etherscan, I like it. 👍

Reply:
Better yet: `seth source-source`

Reply:
So I guess the name of this command is actually a pun @asymmetric, hence the confusion (`source` seems to be used as a verb, but the verb meaning of `source` means "to find something" and doesn't actually imply that the thing that you are looking for is, a "source").

Reply:
Right right, it’s just that it doesn’t make much sense to me to “source a smart contract’s (source) code”, but I might be wrong and also I don’t really actually care ;)

Fine for me if it’s fine for you

Reply:
it's also not that different from `seth tx` and `seth block`

Reply:
That's true